client/components/userForm.js

```diff
@@ -0,0 +1,88 @@
+import React, {Component} from 'react'
+import {Form} from 'semantic-ui-react'
+import {connect} from 'react-redux'
+import {newUser} from '../store/user'
+
+class UserForm extends Component {
+  constructor() {
+    super()
+    this.state = {
+      userName: '',
+      firstName: '',
+      lastName: '',
+      email: '',
+      password: ''
+    }
+    this.handleChange = this.handleChange.bind(this)
```

Comment:
Same here. These are arrow functions, so they are already in the proper context.

Reply:
Everything else looks good